nanit/deep-high-resolution-net.pytorch
# Add coordinates decoding encoding (DARK)

# Intro
  - Add coordinates decoding encoding (DARK)
  - Based on https://arxiv.org/pdf/1910.06278.pdf "Distribution-Aware Coordinate Representation for Human Pose Estimation"
  - Improve PCKh@0.1 results by ~0.6%
# Project
  - Skeleton
# Implemented Sections
  - Generating Target heatmaps
  - Decoding prediction to joint coordinates
  - Did it require refactoring in other locations as well? No
# Tests Description
  - Run Training verify the improvement is not noise
  - Export Model run inference with torchserve
  - Run "Algorithms-Run-Evaluations" s3://nanit-algo-resources/smart-sheets/evaluation_results/2022_03_30/
  - 
# Undeveloped Sections
  - 
# What's New
  - New way to encode / decode coordinates
# Checklist
  - [x] Module tested fully after last commit
  - [ ] Scheduled a PR meeting (if needed)
  - [ ] Documentation (comments or a dedicated document)
### Tested platforms
  - [ ] Cloud (SageMaker)
### Tested Cameras
  - [ ] Gen 1.5
  - [ ] Gen 2
# Comments
### Included Features
  - 
### Infrastructure Dependencies
  - 
### Tech Comments
  - https://wandb.ai/algo/auto-skeleton/runs/22nf2j05
### Testing Instructions
  - 
### Todos
  - 

![](https://nanit.com/favicon.ico)
